Products/ZenUI3/browser/resources/js/zenoss/form/graphPanel.js

```diff
@@ -469,7 +469,7 @@
                 encodedConfig, link,
                 drange="null",
                 // keys to exclude when cloning config object
-                exclusions = ["dockedItems", 'html', 'id', 'tbar'];
+                exclusions = ["dockedItems", 'html', 'id', 'tbar', 'listeners'];
```

Comment:
`listeners` property has a circular reference and this cases an error when config object is converted to string using `Ext.JSON.encode`